scintillometry/utils/phase_utils.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import warnings
+
+import numpy as np
+import astropy.units as u
+import astropy.time as time
```

Comment:
Is `time` used at all? I think it is not such a good idea to import as `time`, since we use the argument `time` quite often.  I would do `from astropy.time import Time` (here if needed, everywhere below)